README.md

```diff
@@ -34,6 +34,8 @@ $ kinesis-console-consumer --help
     --type-at <sequence_number>     start reading from this sequence number (AT_SEQUENCE_NUMBER)
     --type-after <sequence_number>  start reading after this sequence number (AFTER_SEQUENCE_NUMBER)
     --type-timestamp <timestamp>    start reading after this time (units: epoch seconds) (AT_TIMESTAMP)
+    --new-line                      print each record to a new line
+    --regex-filter <regexFilter>    filter data using this regular expression
```

Comment:
I'm not sure about the value of a regex filter when there's already `grep` on the command line. Users of this would have to know that their regex would need to be JavaScript. While `grep`, while it could be considered arcane, is more widely adopted and known.

For example:
```
kinesis-console-consumer foo-stream --regex-filter 'thing'
```
vs
```
kinesis-console-consumer foo-stream | grep 'thing'
```

I'll try to come up with an example where there could be confusion passing in a JS regex from the command line.

Reply:
Yes, grep is amazing... if you're on Linux or Mac.